www/pesa06/cv02/index.php

```diff
@@ -10,5 +10,5 @@
 ?>
 
 <?php include 'header.html';?>
+<?php include 'scripts.html';?>
```

Comment:
ma to byt pred `</body>` a ne pred `<body>`

Reply:
upraveno